src/component/Pages/HomePage.jsx

```diff
@@ -1,99 +1,115 @@
 import React from 'react';
 import Rating from '../ui/Rating/Rating';
 import Button from '../ui/buttons/Button';
+// import BookmarksCards from '../ui/cards/cards';
 import { DemoArticleCoponentWithGlowState } from '../ui/buttons/glow/DemoArticleCoponentWithGlowState';
+import ArticleCardVertical from '../ui/cards/BookmarksCards';
```

Comment:
It seems like your importing a default export from BookmarksCard. Shouldn't it match the file name? Please look into this.